### DISCUSSION TITLE
Possibility to switch off Plugin Tray

### DISCUSSION THREAD

**Discussion by Pete3030** [apohl79/audiogridder#697] [Help]
Hi there,
Is there maybe a possibility to deactivate the plugin tray completely? My mac always asks if I want to accept incoming network connection to plugin tray, although I already blocked that in my firewall. Even when I am currently bouncing a track those messages appear and my cpu spikes which renders the bounce useless. I tried to delete the process (plugin tray) via activity monitor but it still activates itself again. Even If I allow connections plugin tray switches itself on and off during a (online) bounce which causes the cpu to spike (and renders the bounce useless).. Any help would be highly appreciated..

Regards,
Peter

**Comment by apohl79** [comment]
If you manually allow incoming connections for the tray app, does this work? Blocking is the opposite of what the app needs. Normally you just click "yes" in the popup asking if incoming connections are allowed, and it works. The plugins and the trayapp communicate over local network connections.

**Reply by Pete3030** [reply]
Yes I did that, but the problem is that during a online bounce, the plugin tray app logo dissapears sometimes and then it seems to reconnect and appears again, so it somehow seems to loose connection. While reconnecting the cpu Spikes happen and this interferes with the bounce. Would it be possible to just delete the app from the program folder? As I really do not need the functionality of it for my own purposes.. or maybe you don’t allow reconnections during bounce on the code level, if this is possible..

**Comment by Pete3030** [comment]
OK, I just tried it, deleting the Plugin Tray is not possible, otherwise AG plugin does not start. I think it would be good to have an option to switch it off, or (preferably) to get it to runs stable. In my case the network itself runs stable (if there would be a problem the other overview window would appear until reconnection is done automatically), seems that only the Plugin Tray app sometimes losses connection. What I also found strange, that always messages appeared which ask my to grant incoming network connection permission although I had that already accepted in the firewall options and that my /cores folder had a lot data in it (several gb), which was not the case before with V1.1, I already had this problem in a previous beta prior to V1.1 (i think it was related to the plugin scanning process at that time..). hope that helps..;)

**Comment by Pete3030** [comment]
Update: deleting the files in the /cores folder seemed to help, no more hiccups in todays bounce. But I had during the session on spinning ball, which in return seemed to result in another 6gb file in the cores folder (for me it seems like these 6gb are written when the spinning ball is active, then after a few seconds of spinning ball Logic quit unexpectedly..

**Reply by apohl79** [reply]
Try to disable coredumps in the plugin settings under diagnostics. This is only available in the latest beta.

**Comment by Pete3030** [comment]
OK great, I will try that. Thanks!
